docs/developer/rest_api_doc.md

```diff
@@ -69,7 +69,7 @@ filtered and produces an error, if found.
 suite. To manually check the generated spec file, run
 
 ```bash
-rucio/tools/test/check_rest_api_documentation.sh FILE
+rucio_documentation/tools/check_rest_api_documentation.sh FILE
```

Comment:
Maybe we should say this to be explicit

```suggestion
[rucio documentation folder]/tools/check_rest_api_documentation.sh FILE
```

Reply:
If we're linking directly to the file anyways, is there any harm in going Full explicit and doing
```suggestion
[rucio_documentation](https://github.com/rucio/documentation)/tools/check_rest_api_documentation.sh FILE
```
?

Reply:
I like the idea, but I feel that the `/` in `rucio/documentation` could be interpreted as being part of the path, so it might be confusing